actix-http/src/cloneable.rs

```diff
@@ -10,7 +10,7 @@ use actix_service::Service;
 /// # Panics
 /// CloneableService might panic with some creative use of thread local storage.
 /// See https://github.com/actix/actix-web/issues/1295 for example
-pub(crate) struct CloneableService<T: Service>(Rc<RefCell<T>>);
+pub struct CloneableService<T: Service>(Rc<RefCell<T>>);
```

Comment:
I'm of the opinion that it's easier to grok the scope of these when they're pub(crate) vs pub without a lib re-export.

Reply:
I has to be public right? As HttpServiceHandlerResponse is now a public enum